### DISCUSSION TITLE
Pips are disappearing

### DISCUSSION THREAD

**Discussion by sonmezcem** [simeydotme/svelte-range-slider-pips#141] [Formatting / Styling] [answered]
Hi,

It is bug or not, I am not sure. I am using Vanilla JS version. The issue is when I set range: true. The pips are disappearing but it seems like it is there. To clarify, please check the screen shot.

![ss](https://github.com/simeydotme/svelte-range-slider-pips/assets/117642309/039e30f7-cef1-4ef0-8459-e58550b50298)

**Comment by sonmezcem** [comment, marked as answer]
The solution was awaiting my question 👍 , I have added .rangeSlider overflow: visible and the pips are back.

`.rangeSlider {
    overflow: visible;
}
`

**Reply by simeydotme** [reply]
haha glad you sorted it!

**Reply by sonmezcem** [reply]
thank you

**Comment by simeydotme** [comment]
By the way, thanks for using discussion and not opening an issue! ;D

And I have one question;
**How/Why did you decide on using this library for a Vanilla project?** ~ I ask because the project has the name Svelte in it. Although I always planned to export it as a standalone library too, and glad you are using it! I was torn if I should include the name 'svelte' in the package name because I didn't want to scare people away who were not using svelte.

**Reply by sonmezcem** [reply]
You are welcome.

I am actually not an experienced developer. So, I am not familiar with Svelte. It seemed easier to use Vanilla JS because I have some knowledge Vanilla JS from past experiences. So that's the reason I decided to use it in a Vanilla project.
